src/Http/Controllers/CP/Users/PasswordController.php

```diff
@@ -17,7 +17,7 @@ public function update(Request $request, $user)
     {
         throw_unless($user = User::find($user), new NotFoundHttpException);
 
-        $updatingOwnPassword = $user->id() == $request->user()->id();
+        $updatingOwnPassword = $user->id() == User::fromUser($request->user())->id();
```

Comment:
I think we might be able to call `->getKey()` on the user, instead of `->id()`, like this:

```suggestion
        $updatingOwnPassword = $user->id() == $request->user()->getKey();
```

Reply:
Happy to change this - though I was following convention I could see elsewhere in the codebase where 
`$request->user()`
had been wrapped in the same way as here.

Reply:
The User::fromUser($user)->id() what you've done is good 👍